sympy/matrices/matrices.py

```diff
@@ -302,11 +302,46 @@ def copy(self):
         return self._new(self.rows, self.cols, self._mat)
 
     def trace(self):
+        """
+        Represents the trace of a matrix expression.
+
+        Examples
+        ========
+
+        >>> from sympy import MatrixSymbol, Trace, eye
+        >>> A = MatrixSymbol('A', 3, 3)
+        >>> Trace(A)
+        Trace(A)
+        >>> Trace(eye(3))
+        Trace(Matrix([[1,0,0],[0,1,0],[0,0,1]]))
+
+        """
         if not self.is_square:
             raise NonSquareMatrixError()
         return self._eval_trace()
 
     def inv(self, method=None, **kwargs):
+        """
+        Returns the inverse of the matrix `K`, if it exists using the Gaussian elimination by default.
+        However we can specify it to be done by LU decomposition as well.
```

Comment:
Returns the inverse of a matrix , using Gaussian elimination by default. By setting ``method='LU'``  we can specify it to be done using LU decomposition.